gists/snippets/react.md

```diff
@@ -117,6 +117,52 @@ Copy the following into your `javascript.json` snippet file:
 			")(${1:MyFormComponent})"
 		],
 		"description": "LP-Form enhanced Redux-Form form component"
+	},
+
+	"React Formik Component": {
+		"prefix": "reactFormikComponent",
+		"body": [
+			"import React from 'react'",
+			"import PropTypes from 'prop-types'",
+			"import { Formik, Form } from 'formik'",
+			"import * as Yup from 'yup'",
+			"import { SubmitButton } from 'lp-components'",
+			"",
+			"const propTypes = {",
+			"\thandleSubmit: PropTypes.func.isRequired,",
+			"}",
+			"const defaultProps = {}",
+			"",
+			"const BASE_INITIAL_VALUES = {",
+			"}",
```

Comment:
It's a little wonky (imo) to have `const defaultProps = {}` on one line, but
```
const BASE_INITIAL_VALUES = {
}
```
on two

Reply:
Good point Agree and fixed.